lib/login_gov/hostdata.rb

```diff
@@ -28,6 +30,23 @@ def self.in_datacenter?
       File.directory?(CONFIG_DIR)
     end
 
+    # @yield Executes a block if in_datacenter?
+    # @yieldparam hostdata
+    def self.in_datacenter
+      raise LocalJumpError, 'in_datacenter must be called with a block' unless block_given?
+      yield self if in_datacenter?
+    end
+
+    # @return [S3]
+    def self.s3
+      ec2 = LoginGov::Hostdata::EC2.load
+
+      LoginGov::Hostdata::S3.new(
+        env: env,
+        bucket: "login-gov-app-secrets-#{ec2.region}-#{ec2.account_id}",
```

Comment:
@brodygov -- is this ok to hardcode here? it simplifies calling code a bunch, and I think our plan is to have like `/idp` directories for the IDP etc etc, so it should be fine?

Reply:
Yeah it seems fine to hardcode this.

The names are generated as `login-gov.app-secrets.<ACCOUNT_ID>-<REGION>`.